geode-protobuf-messages/src/main/proto/v1/connection_API.proto

```diff
@@ -23,3 +23,11 @@ message AuthenticationRequest {
 message AuthenticationResponse {
     bool authenticated = 1;
 }
+
+message DisconnectClientRequest {
+    string reason = 1;
+}
+
+message DisconnectClientResponse {
+    bool acknowledged = 1;
```

Comment:
Would we ever send this response with this field equal to false?  If not we should remove it.

Reply:
Whilst we could have an empty response, I was hedging my bets in case there was pending work for that client. Maybe that's too paranoid.